encoder_test.go

```diff
@@ -103,6 +103,18 @@ func TestEncodeDecodeSuccess(t *testing.T) {
 				"È",
 			},
 		},
+		{
+			testCase: TestCase{Name: "upper-case quotes"},
+			tokens: []string{
+				"O", "'", "SH", "E", "AN", " DON", "'", "T", " BE", " BA", "'", "D", " '", "MAN", " YOU", "'", "RE", " CO", "ULD", "'", "VE", " HE", "'", "L", "LP", "ED",
+			},
+		},
+		{
+			testCase: TestCase{Name: "lower-case quotes"},
```

Comment:
i don't follow description of the test, what does "quote" refer to here? Also does the test fail without your change?